### PR TITLE
[bugfix] recomputeNavMesh with stage config

### PR DESCRIPTION
**Pull request by aclegg3**
## Motivation and Context

`Simulator::recomputeNavMesh` was using the `SimulatorConfiguration.scene.id` field to find the render asset for the stage. This could now be a json path or template handle. Instead, the asset of the stage should be queried from the `RigidStage` created in `PhysicsManager`. 

This change adds a getter for the `StageAttributes` object used to initialize the `RigidStage`, modified `Simulator::recomputeNavMesh`  using the initialization attributes, and handling for the `"NONE"` stage.

## How Has This Been Tested

Tested on both old-style stage asset loading and loading stage from config file with and without `--enable-physics` and `"NONE"` stage.

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/esp/physics/PhysicsManager.h [hunk 1]
```diff
@@ -871,6 +871,16 @@ class PhysicsManager {
...
 
+  /**
+   * @brief Get a copy of the template used to initialize the stage.
+   *
+   * @return The initialization settings of the stage or nullptr if the stage is
+   * not initialized.
+   */
+  Attrs::StageAttributes::ptr getStageInitAttributes() const {
+    return staticStageObject_->getInitializationAttributes();
+  }
```
Comment (jturner65) [inline src/esp/physics/PhysicsManager.h:882]: Maybe want to py-bind this like we do for objects?

**Review by jturner65 — APPROVED** [review]
Small comment.  Otherwise looks good for now, until we figure out the particulars of NONE stage.

**Review by jturner65 — APPROVED** [review]
LGTM

File: src/esp/physics/RigidBase.h [hunk 2]
```diff
@@ -590,10 +590,12 @@ class RigidBase : public Magnum::SceneGraph::AbstractFeature3D {
    * @brief Get a copy of the template used to initialize this object
    * or scene.
    * @return A copy of the initialization template used to create this object
-   * instance.
+   * instance or nullptr if no template exists.
    */
   template <class T>
   std::shared_ptr<T> getInitializationAttributes() const {
+    if (initializationAttributes_ == nullptr)
```
Comment (bigbike) [inline src/esp/physics/RigidBase.h:597]:
```
if (!initializationAttributes_) {
  return nullptr;
}
```

File: src/esp/sim/Simulator.cpp [hunk 3]
```diff
@@ -606,8 +614,12 @@ bool Simulator::recomputeNavMesh(nav::PathFinder& pathfinder,
                  "loaded without renderer initialization.",
                  false);
 
-  assets::MeshData::uptr joinedMesh =
-      resourceManager_->createJoinedCollisionMesh(config_.scene.id);
+  assets::MeshData::uptr joinedMesh = assets::MeshData::create_unique();
```
Comment (bigbike) [inline src/esp/sim/Simulator.cpp:617]: Make sure there is some initial value here (e.g., nullptr).

Comment (aclegg3) [reply src/esp/sim/Simulator.cpp:617]: The new object is the initial value. More verts/indices are added by STATIC non-stage objects.

**Review by Skylion007 — APPROVED** [review]

**Pull request merged**